source/documentation/virtual_machine_management_guide/topics/Optimizing_Live_Migration.adoc

```diff
@@ -0,0 +1,55 @@
+[[Optimizing_Live_Migration]]
+==== Optimizing Live Migration
+
+Live virtual machine migration can be a resource-intensive operation. To optimize live migration, you can set the following two options globally for every virtual machine in an environment, for every virtual machine in a cluster, or for an individual virtual machine.
+
+[NOTE]
+====
+For cluster levels 4.3 or later, auto converge is enabled by default for all built-in migration policies, and migration compression is enabled by default for only the `Suspend workload if needed` migration policy. You can change these parameters when adding a new migration policy, or by modifying the `MigrationPolicies` configuration value.
+====
+
+The *Auto Converge migrations* option allows you to set whether auto-convergence is used during live migration of virtual machines. Large virtual machines with high workloads can dirty memory more quickly than the transfer rate achieved during live migration, and prevent the migration from converging. Auto-convergence capabilities in QEMU allow you to force convergence of virtual machine migrations. QEMU automatically detects a lack of convergence and triggers a throttle-down of the vCPUs on the virtual machine.
+
+The *Enable migration compression* option allows you to set whether migration compression is used during live migration of the virtual machine. This feature uses Xor Binary Zero Run-Length-Encoding to reduce virtual machine downtime and total live migration time for virtual machines running memory write-intensive workloads or for any application with a sparse memory update pattern.
+
+Both options are disabled globally by default.
+
+
+*Configuring Auto-convergence and Migration Compression for Virtual Machine Migration*
+
+. Configure the optimization settings at the global level:
+.. Enable auto-convergence at the global level:
```

Comment:
I'm not sure that it makes sense to make this a sub-procedure. Why not just make each of these steps part of the main procedure? That would leave with a procedure of 6 steps, which is certainly an acceptable length.